wcivf/apps/people/templates/people/not_current_person_detail.html

```diff
@@ -18,6 +18,7 @@ <h2 class="person_name">{{ object.name }}
       (Deceased)
     {% endif %}
     </h2>
+    <p>{{ object.name }} stood for election {{ object.past_not_current_candidacies.count }} times.</p>
```

Comment:
I think this is useful, but we need to think about how to explain that we've only been tracking local elections since 2016 and generals since 2010. (and by-elections since ~2018). In other words, this is actually the number of elections we've tracked this person for. Help from @pmk01 welcome.

Reply:
Yes, this is an interesting one. We also don't do parish elections, which some people might also be confused by.

We could say something like :

 [candidate] has stood in [n] elections collected in the Democracy Club database*.
or
According to our database*, [candidate] has stood in [n] elections since 2010

*perhaps 'Democracy Club database' could link to a separate page on WCIVF which explains the history of the project in a couple of sentences? Awkward perhaps, but it would be useful for us to have it all explained concisely somewhere.

Reply:
All good ideas. For simplicity, I suggest:

[candidate] has stood in [n] elections since 2010*

*link to explainer as suggested by @pmk01 or the summary given by @symroe